### PR TITLE
[CodePush] Implemented data contracts.

### PR DESCRIPTION
**Pull request by annakocheshkova**
Implemented all the data contracts for the project + covered with tests.

### CODE REVIEW

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/datacontracts/CodePushDownloadStatusReport.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        codePushDownloadStatusReport.setClientUniqueId(clientUniqueId);
+        codePushDownloadStatusReport.setDeploymentKey(deploymentKey);
+        codePushDownloadStatusReport.setLabel(label);
+        return codePushDownloadStatusReport;
+    }
+
+    /**
+     * Gets the value of clientUniqueId and returns it.
+     *
+     * @return clientUniqueId.
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/datacontracts/CodePushDownloadStatusReport.java]: I guess _meaning_ be better than a name of var in comments.

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushTest.java [hunk 2]
```diff
@@ -12,26 +22,191 @@
...
         CodePushUpdateState codePushUpdateState = CodePushUpdateState.LATEST;
         int updateStateValue = codePushUpdateState.getValue();
-        assertEquals(updateStateValue, 2);
+        assertEquals(2, updateStateValue);
+    }
+
+    @Test
+    public void dataContractsTest() throws Exception {
+
+        /*Check download report. */
```
Comment (MatkovIvan) [inline sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushTest.java]: Space after `/*`

File: sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/datacontracts/CodePushDownloadStatusReport.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+     */
+    @SerializedName("label")
+    private String label;
+
+    /**
+     * Creates a report using the provided information.
+     *
+     * @param clientUniqueId id of the device.
+     * @param deploymentKey  deployment key to use to query the CodePush server for an update.
+     * @param label          internal label automatically given to the update by the CodePush server.
```
Comment (sergey-akhalkov) [inline sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/datacontracts/CodePushDownloadStatusReport.java]: Extra spaces

Comment (MatkovIvan) [reply sdk/appcenter-codepush/src/main/java/com/microsoft/appcenter/codepush/datacontracts/CodePushDownloadStatusReport.java]: @sergey-akhalkov Alignment here is OK

File: sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushTest.java [hunk 4]
```diff
@@ -12,26 +22,191 @@
...
+        assertEquals(6, syncStatusValue);
 
         CodePushUpdateState codePushUpdateState = CodePushUpdateState.LATEST;
         int updateStateValue = codePushUpdateState.getValue();
-        assertEquals(updateStateValue, 2);
+        assertEquals(2, updateStateValue);
+    }
+
+    @Test
+    public void dataContractsTest() throws Exception {
```
Comment (ruslan-bikkinin) [inline sdk/appcenter-codepush/src/test/java/com/microsoft/appcenter/codepush/CodePushTest.java]: I believe, each data contract should have its own separated test file.

**Pull request closed without merging**